client/tool/source/Main.cpp

```diff
@@ -966,6 +966,55 @@ static void shutdownCommand(const std::shared_ptr<IDobbyProxy>& dobbyProxy,
     AI_LOG_FN_EXIT();
 }
 
+// -----------------------------------------------------------------------------
+/**
+ * @brief
+ *
+ *
+ *
+ */
+static void setLogLevelCommand(const std::shared_ptr<IDobbyProxy>& dobbyProxy,
+                               const std::shared_ptr<const IReadLineContext>& readLine,
+                               const std::vector<std::string>& args)
+{
+    AI_LOG_FN_ENTRY();
+
+    if (args.size() < 1 || args[0].empty())
+    {
+        readLine->printLnError("must provide at least one arg; <level>");
+        return;
+    }
+
+    const std::string level = args[0];
+
+    int levelNo;
+    if (strcasecmp(level.c_str(), "FATAL") == 0)
+        levelNo = AI_DEBUG_LEVEL_FATAL;
+    else if (strcasecmp(level.c_str(), "ERROR") == 0)
+        levelNo = AI_DEBUG_LEVEL_ERROR;
+    else if (strcasecmp(level.c_str(), "WARNING") == 0)
+        levelNo = AI_DEBUG_LEVEL_WARNING;
+    else if (strcasecmp(level.c_str(), "MILESTONE") == 0)
+        levelNo = AI_DEBUG_LEVEL_MILESTONE;
+    else if (strcasecmp(level.c_str(), "INFO") == 0)
+        levelNo = AI_DEBUG_LEVEL_INFO;
+    else if (strcasecmp(level.c_str(), "DEBUG") == 0)
+        levelNo = AI_DEBUG_LEVEL_DEBUG;
+    else
+    {
+        readLine->printLnError("Error: invalid LEVEL argument, possible values are "
+                               "FATAL, ERROR, WARNING, MILESTONE, INFO or DEBUG\n");
```

Comment:
printLnError adds new line character by itself, so when you added this one we now have 2 newlines. But I believe that we can live with that.